webcomponents/input/input.webcomponent.spec.js

```diff
@@ -183,6 +183,16 @@ describe('mn-input (webcomponent)', () => {
   })
 
   describe('attribute readonly', () => {
+    it('should define attribute inside of a element', () => {
+      element.setAttribute('readonly','readonly')                     // even no setting value, the attribute readonly is receiving value readonly as value.
```

Comment:
I made a temporary fix, assertion now works, try invert (with/without .not)

I will get the chai-dom and test this type of assertion

```js
// dont work, but should
// input inside element, in this assertion, NOT have attribute, and assertion dont broke
expect(element).to.contain('input').to.have.attribute('readonly')
```

help me please, look in chai-dom and open an issue to us, lets investigate, and contribute with chai-dom if is good